src/estimation/include/iDynTree/Estimation/BerdyHelper.h

```diff
@@ -228,6 +256,12 @@ struct BerdyOptions
      */
     std::string baseLink;
 
+    /**
+     * This is a boolean flag for stack of task MAP estimation
+     * The default value is false
+     */
+    bool stackOfTasksMAP;
```

Comment:
Is this used somewhere?

Reply:
Removed in https://github.com/robotology/idyntree/pull/783/commits/4d0f4da1abc9a7a416414aecad372f0fbcc5e054